src/solver/optimisation/opt_calcul_des_pmin_MUT_MDT.cpp

```diff
@@ -44,51 +44,50 @@ double OPT_CalculerAireMaxPminJour(int PremierPdt,
                                    int* NbGrpCourbeGuide,
                                    int* NbGrpOpt)
 {
-    int T;
-    int i;
-
     double Cout = 0.0;
     int NbMx = 0;
 
-    for (T = 0; T < PremierPdt; T++)
+    for (int i = 0; i < PremierPdt; i++)
     {
-        if (NbGrpCourbeGuide[T] > NbMx)
-            NbMx = NbGrpCourbeGuide[T];
+        if (NbGrpCourbeGuide[i] > NbMx)
+            NbMx = NbGrpCourbeGuide[i];
     }
 
-    for (T = DernierPdt; T < NombreDePasDeTemps; T++)
+    for (int i = DernierPdt; i < NombreDePasDeTemps; i++)
     {
-        if (NbGrpCourbeGuide[T] > NbMx)
-            NbMx = NbGrpCourbeGuide[T];
+        if (NbGrpCourbeGuide[i] > NbMx)
+            NbMx = NbGrpCourbeGuide[i];
     }
 
-    for (T = 0; T < PremierPdt; T++)
+    for (int i = 0; i < PremierPdt; i++)
     {
-        NbGrpOpt[T] = NbMx;
-        Cout += (double)(NbGrpOpt[T] - NbGrpCourbeGuide[T]);
+        NbGrpOpt[i] = NbMx;
+        Cout += (double)(NbGrpOpt[i] - NbGrpCourbeGuide[i]);
     }
 
-    for (T = DernierPdt; T < NombreDePasDeTemps; T++)
+    for (int i = DernierPdt; i < NombreDePasDeTemps; i++)
     {
-        NbGrpOpt[T] = NbMx;
-        Cout += (double)(NbGrpOpt[T] - NbGrpCourbeGuide[T]);
+        NbGrpOpt[i] = NbMx;
+        Cout += (double)(NbGrpOpt[i] - NbGrpCourbeGuide[i]);
     }
 
-    T = PremierPdt;
-    while (T < DernierPdt)
+    int i = PremierPdt;
+    while (i < DernierPdt)
     {
         NbMx = 0;
-        for (i = 0; i < MUTetMDT && T < DernierPdt; i++, T++)
+        int tmp = 0;
+        for (int j = 0; j < MUTetMDT && i < DernierPdt; j++, i++)
         {
-            if (NbGrpCourbeGuide[T] > NbMx)
-                NbMx = NbGrpCourbeGuide[T];
+            if (NbGrpCourbeGuide[i] > NbMx)
+                NbMx = NbGrpCourbeGuide[i];
+            tmp = j;
         }
 
-        T -= i;
-        for (i = 0; i < MUTetMDT && T < DernierPdt; i++, T++)
+        i -= tmp;
```

Comment:
You should make an exception here and declare j outside, with a more expressive name